script/postrelease

```diff
@@ -12,9 +12,6 @@ puts "Enter the version to deploy to prod"
 puts
 npm_version = STDIN.gets.strip
 puts "Are you sure to deploy '#{npm_version}' this to prod?"
-puts
-puts "This will reset the HEAD to origin/master?"
-puts
 puts "Current staged and local changes will be lost!"
```

Comment:
This line also could be removed as we are not resetting now. Sorry should have catched it first.